generate-hostcert

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import json
+
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.backends import default_backend
+
+from cryptography import x509
+from cryptography.x509.oid import NameOID
+from cryptography.hazmat.primitives import hashes
+
+import requests
+
+# First, generate a new host key.
+key = rsa.generate_private_key(
+    public_exponent=65537,
+    key_size=2048,
+    backend=default_backend(),
+)
+
+key_bytes = key.private_bytes(
+    encoding=serialization.Encoding.PEM,
+    format=serialization.PrivateFormat.TraditionalOpenSSL,
+    encryption_algorithm=serialization.NoEncryption()
+)
+
+csr = x509.CertificateSigningRequestBuilder(
+   ).subject_name(x509.Name([x509.NameAttribute(NameOID.COMMON_NAME, u"will_be_overwritten")])
+   ).sign(key, hashes.SHA256(), backend=default_backend())
+
+
+csr_bytes = csr.public_bytes(serialization.Encoding.PEM).decode()
+
+with open(sys.argv[1], "rb") as fp:
+    token = fp.read().strip()
+
+try:
+    response = requests.post("https://os-registry.osgdev.chtc.io/syslog-ca/issue", data={"csr": csr_bytes},
```

Comment:
```suggestion
    response = requests.post("https://os-registry.opensciencegrid.org/syslog-ca/issue", data={"csr": csr_bytes},
```

Once we deploy the new registry to prod, we should point to it

Reply:
I don't think we have the opensciencegrid.org alias so we should change this to `osg.chtc.io` for now

Reply:
Hrm,we're getting it from somewhere! https://os-registry.opensciencegrid.org/

Reply:
Never mind, I was thinking of the syslog pod.

Reply:
New registry is up.